app/src/main/java/com/yterletskyi/happyfriend/features/contacts/ui/ContactsFragment.kt

```diff
@@ -51,6 +51,23 @@ class ContactsFragment : BaseBindingFragment<FragmentContactsBinding>(
             }
         }
 
+        with(binding.etSearch) {
+            binding.etClear.visibility = View.INVISIBLE
+            doAfterTextChanged {
+                if (text?.isEmpty() == false) {
+                    binding.etClear.visibility = View.VISIBLE
+                } else {
+                    binding.etClear.visibility = View.INVISIBLE
```

Comment:
Views with visibility `View.INVISIBLE` can be clicked actually - they just don't show up. Anyway this will not cause a bug for us, but let's use `View.GONE`.